server/src/main/java/org/candlepin/hostedtest/HostedTestSubscriptionResource.java

```diff
@@ -188,6 +188,7 @@ public boolean deleteSubscription(@PathParam("subscription_id") String subscript
      * Deletes all subscriptions.
      */
     @DELETE
+    @Produces(MediaType.APPLICATION_JSON)
```

Comment:
Pretty sure I put this there because we have a servlet context listener (or the like) that prints warnings when there are REST methods without an `@Produces` annotation.  I believe it serves merely to remove a warning when deploying in hosted mode.  I'll double check and make sure it's truly necessary.